lib/OpenQA/Setup.pm

```diff
@@ -96,6 +96,7 @@ sub read_config ($app) {
             token_label => '',
             nickname_from => '',
             unique_name => '',
+            id_from => 'id',
```

Comment:
can you add the according config option also to etc/openqa.ini with an explanatory comment from your git commit message?

Reply:
It looks like `t/config.t` also needs to be adjusted.

Reply:
All the test issues should be resolved now. Also added the comment to `etc/openqa.ini`. Thanks for the reviews